### ISSUE TITLE
Modular javascript and the option to trigger/reinitialize!

### ISSUE THREAD

**Issue by fourpixels** [Dogfalo/materialize#989]
Right now I struggle with forms. I have a simple input field, and everything works out great. Until I create some input element with code and append it to DOM. In your code, there is this piece:

```js
// Add active if input element has been pre-populated on document ready
    $(document).ready(function() {
      $(input_selector).each(function(index, element) {
        if($(element).val().length > 0) {
          $(this).siblings('label, i').addClass('active');
        }
      });
    });
```

It's nice, but it runs **only once**, because the whole forms plugin (as well as all other plugins) are initialized one time **only!** So now I have to copy-paste that code and re-run it every time I attach something to the DOM, which is killing me.

In my opinion, you should think of modular way to provide plugins so that they can be initialized/refreshed once one needs to.

p.s.
Forgot to mention that I already did this with `collapsible` and call it each time I add an item (facepalm)

**Comment by Dogfalo** [comment]
duplicate of https://github.com/Dogfalo/materialize/issues/250
